flink-core/src/main/java/org/apache/flink/configuration/MetricOptions.java

```diff
@@ -25,20 +25,9 @@
 public class MetricOptions {
 
 	/**
-	 * The list of named reporters. Names are defined here and per-reporter configs
-	 * are given with the reporter config prefix and the reporter name.
-	 *
-	 * Example:
-	 * <pre>{@code
-	 * metrics.reporters = foo, bar
-	 *
-	 * metrics.reporter.foo.class = org.apache.flink.metrics.reporter.JMXReporter
-	 * metrics.reporter.foo.interval = 10
-	 *
-	 * metrics.reporter.bar.class = org.apache.flink.metrics.graphite.GraphiteReporter
-	 * metrics.reporter.bar.port = 1337
-	 * }</pre>
+	 * @deprecated This configuration key has no effect.
 	 */
+	@Deprecated
```

Comment:
Have we removed all usages of this key? Can we simply delete the option? The behavior of any reporters relying on this field will be broken so is it better to change the `@PublicEvolving` API?

Reply:
This option should only be used by Flink in the setup of reporters, and tests/user-code to configure reporters. All these cases should continue to work even with the option being a no-op. The only case that may now fail if some test was configuring this option but no actual reporter, but well, that will fail regardless of whether we keep the option or not.

I'd rather keep it to not cause compilation failures.